lcls2-cc-lib/Library/Devices/SLITS/FB_SLITS.TcPOU

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_SLITS" Id="{e16e0f37-b04b-4e2b-a3f3-e910fdc91095}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_SLITS
+VAR_IN_OUT
+
+END_VAR
+VAR_INPUT
+
+END_VAR
+VAR_OUTPUT
+END_VAR
+VAR
+
+
+
+END_VAR
```

Comment:
Should clean up these empty declaration blocks if they aren't being used

Reply:
Done.